tests/lib/rules/self-closing-comp.js

```diff
@@ -37,6 +37,10 @@ ruleTester.run('self-closing-comp', rule, {
           <Hello name="John" />
         </Hello>
       `
+    }, {
+      code: 'var HelloJohn = <Hello name="John"> </Hello>;'
+    }, {
+      code: 'var HelloJohn = <Hello name="John">        </Hello>;'
```

Comment:
this does get collapsed to a single space, though.

Reply:
I tried this on jsfiddle and didn't find it collapsed.

Reply:
hm, it should - the browser itself does that.

Reply:
Yay I mean react rendered html string does not collapse spaces in this case. Browser does.